ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/vxlan_interface.py

```diff
@@ -132,10 +132,17 @@ def _get_vxlan_interface_config_for_vrf(self, vrf: dict, tenant: dict, vrfs: lis
             if "evpn" not in vrf.get("address_families", ["evpn"]):
                 return
 
-            vni = default(
-                vrf.get("vrf_vni"),
-                vrf.get("vrf_id"),
-            )
+            if self.shared_utils.wan_role is not None:
+                # vni = get(vrf, "wan_vni", required=True, org_key=f"VRF {vrf_name} in tenant {tenant['name']} does not have a `wan_vni` defined.")
```

Comment:
keep error message but make it clearer

TODO - if this VRF was not intended to be extended over the WAN, set address family, if not intended on the WAN router use the filer VRF (todo ;) )